src/ml/neural_net/model_spec.cpp

```diff
@@ -586,6 +589,8 @@ void model_spec::add_pooling(const std::string& name, const std::string& input,
     case padding_type::SAME:
       params->mutable_same();
       break;
+    case padding_type::REFLECTIVE:
+      break;
```

Comment:
I believe it defaults to valid?

```
If you set a oneof field to the default value (such as setting an int32 oneof field to 0), the "case" of that oneof field will be set, and the value will be serialized on the wire.
```

https://developers.google.com/protocol-buffers/docs/proto#oneof-features

Reply:
But the point is that you're not setting the field to a default value, you're not setting anything at all. My point is, in what sense does this implement "reflective" padding?

Reply:
I think I remember now the padding available to us in:

- `PoolingPadding`
- `ConvolutionalLayerParams`

have different padding values than:

- `PaddingLayerParams`

 I was trying to use the same enumerated types for different layers, but using the same enum wouldn't work for all types of padding.

Right now since we only handle reflective padding in our MLModels specifically in the `PaddingLayer`. I'll remove the enumerated type and add a `TODO` there for future reference in any of the other models we have specifically have other types of padding in their `PaddingLayer`.